test-utils/pom.xml

```diff
@@ -18,6 +18,7 @@
 
     <properties>
         <hibernate-validator-parent.path>..</hibernate-validator-parent.path>
+        <automatic.module.name>org.hibernate.validator.testutil</automatic.module.name>
```

Comment:
I would choose to be consistent with the artifact name instead of the package name and use testutils with a trailing s.